### PR TITLE
[WOR-646] Add cloudPlatform to workspace details endpoint

### PR DESCRIPTION
**Pull request by ghost**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-646 
- added cloudPlatform field to workspace details  

getWorkspaceById, listWorkspaceDetails now show the cloudPlatform field:
Screen Shot 2022-12-07 at 10 14 08 AM 

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

**Comment by ghost** [comment]
jenkins retest

File: core/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -5153,6 +5153,9 @@ components:
           description: The date all files were successfully transferred to the workspace bucket if applicable
             in yyyy-MM-ddTHH:mm:ss.SSSZZ format
           format: date-time
+        cloudPlatform:
+          type: string
+          description: The cloud platform of the workspace
```
Comment (cahrens) [inline core/src/main/resources/swagger/api-docs.yaml:5158]: After this is merged, remember to update orchestration's swagger.

File: core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala [hunk 2]
```diff
@@ -267,6 +268,23 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
     args
   }
 
+  def getCloudPlatform(workspace: Workspace): Option[WorkspaceCloudPlatform] = {
```
Comment (cahrens) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala:271]: FYI @ehigham as you have been considering a similar helper method.

**Review by cahrens — APPROVED** [review]
Looks straightforward.

**Review by marctalbott — APPROVED** [review]

**Pull request merged**